.github/workflows/README.md

```diff
@@ -50,8 +117,6 @@ This workflow is built from multiple composite actions listed below:
 
 - [helm-lint](https://github.com/bakdata/ci-templates/tree/main/actions/helm-lint)
 - [bump-version](https://github.com/bakdata/ci-templates/tree/main/actions/bump-version)
-- [helm-package](https://github.com/bakdata/ci-templates/tree/main/actions/helm-package)
-- [commit-and-push](https://github.com/bakdata/ci-templates/tree/main/actions/commit-and-push)
```

Comment:
why did you remove those?

Reply:
because none of them is used in `Docker Build and Publish`